hw/ip/tlul/generic_dv/xbar_tests.hjson

```diff
@@ -125,6 +125,8 @@
       en_run_modes: ["xbar_mode"]
       uvm_test: xbar_error_test
       uvm_test_seq: xbar_random_vseq
+      // More seeds to cover assert coverage - d_errorChangedNotAccepted_C
+      reseed: 100
```

Comment:
Can the constraints in the sequence be adjusted to hit this a bit more favorably rather than increase seeds? I am ok if that is too hard.